class.jetpack.php

```diff
@@ -5956,14 +5956,16 @@ public static function dns_prefetch( $new_urls = null ) {
 	public function wp_dashboard_setup() {
 		if ( self::is_active() ) {
 			add_action( 'jetpack_dashboard_widget', array( __CLASS__, 'dashboard_widget_footer' ), 999 );
+			$widget_title = __( "Jetpack", 'jetpack' );
```

Comment:
Fixed in https://github.com/Automattic/jetpack/commit/dd62a9cf29c74f7a6d33b980b89cef22d0185f8d